editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.spec.browser2.tsx

```diff
@@ -606,6 +615,8 @@ export var storyboard = (
 )
 `
 
+/* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["expect", "expectElementWithTestIdNotToBeRendered", "expectElementWithTestIdToBeRendered"] }] */
```

Comment:
is this needed? 🧐

Reply:
yes, a lot of tests that check whether controls are rendered for the correct targets don't use `expect` directly, only the custom expect functions